components/src/main/res/layout/editable_list_item.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.motion.widget.MotionLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/motionLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:layoutDescription="@xml/scene_editing_mode"
+    tools:showPaths="true">
+
+    <TextView
+        android:id="@+id/column_one"
+        style="@style/Text.Body"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:gravity="start" />
+
+    <TextView
+        android:id="@+id/column_two"
```

Comment:
Can we give this one a bit of space between the columns? I think probably 16dp (remember to use our hmrc dimen)
On my emulator they get a bit squashed together when the edit button is showing and the text is long.